### DISCUSSION TITLE
documentation for specific version

### DISCUSSION THREAD

**Discussion by javad94** [python-telegram-bot/python-telegram-bot#2280] [Q&A] [answered]
Hi, How can I view documentation for a specific PTB version without the changes introduced in the next versions? For example, I want to view documentation for version 12, but they are now updated for version 13.

**Comment by Poolitzer** [comment, marked as answer]
hey, that is not possible with our documentation. However, it is very easy to build your own local documentation. Clone the repo, install the documentation requirements (`pip install -r docs/requirements-docs.txt`), checkout the release you want, then build the documentation. Its all static html pages so you can just view them in your browser. 

`make -C docs html` or `sphinx-build docs/source docs/build/html`

**Reply by javad94** [reply]
Thank you. Is there any plan to add versioning to documentation?

**Reply by Poolitzer** [reply]
Im not sure read the docs supports this out of the box.

**Reply by Bibo-Joshi** [reply]
They do. we only support the latest stable release & master anyway, though … But as we're now also starting to add the `add in vXY` info, we might consider publishing them. RTD can even add a banner showing a warning that you're on an old versions docs

**Reply by Bibo-Joshi** [reply]
FYI @javad94 at https://python-telegram-bot.readthedocs.io you can now find all versions down to v12

**Reply by Poolitzer** [reply]
So many versions

**Reply by javad94** [reply]
@Bibo-Joshi Thank you very much! that's exactly what I wanted.
